cluster/manifests/ingress-controller/deployment.yaml

```diff
@@ -51,6 +51,7 @@ spec:
         - --load-balancer-type={{ .Cluster.ConfigItems.kube_aws_ingress_default_lb_type }}
 {{end}}
         - --cert-polling-interval={{ .Cluster.ConfigItems.kube_aws_ingress_controller_cert_polling_interval }}
+        - --ingress-api-version=networking.k8s.io/{{ .Cluster.ConfigItems.kube_aws_ingress_controller_ingress_version }}
```

Comment:
There are three values possible
```go
	IngressAPIVersionExtensions       = "extensions/v1beta1"
	IngressAPIVersionNetworking       = "networking.k8s.io/v1beta1"
	IngressAPIVersionNetworkingV1     = "networking.k8s.io/v1"
```
so maybe it would be better to use full name verion as a config value, i.e.
```yaml
{{if eq .Cluster.Environment "production"}}
kube_aws_ingress_controller_ingress_version: "networking.k8s.io/v1beta1"
{{else}}
kube_aws_ingress_controller_ingress_version: "networking.k8s.io/v1"
{{end}}

--ingress-api-version={{ .Cluster.ConfigItems.kube_aws_ingress_controller_ingress_version }}
```